File: Algorithm/programmers/Level1/문자열 다루기 기본.py
def solution(s):
    if len(s) == 4 or len(s) == 6:
        for char in s:
            if char.isnumeric():
                continue
            else:
                return False
    else:
        return False

    return True

def solution(s):
    if len(s) == 4 or len(s) == 6:
        if s.isdigit():
            return True
        else:
            return False
    else:
        return False

File: Algorithm/programmers/Level1/문자열 다루기 기본.js
function solution(s) {
  if (s.length === 4 || s.length === 6) {
    if (Number(s)) return true;
    else return false;
  }
  return false;
}

function solution(s) {
  let tmp = parseInt(s);
  if (s.length === 4 || s.length === 6) {
    if (s == tmp) return true;
  }
  return false;
}
